lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -463,6 +465,7 @@ instance ToText DBLog where
         MsgQuery stmt _ -> stmt
         MsgRun False -> "Running database action - Start"
         MsgRun True -> "Running database action - Finish"
+        MsgWillOpenDB fp -> "Will open db at " <> (maybe "in-memory" T.pack fp)
```

Comment:
:+1: Anything particular you were looking for :thinking: ?

Reply:
I wanted to browse manually to check 1) if pool registration certificates had metadata references 2) if any metadata had been fetched

I believe I was simply missing an `eventually` though.